internal/extsvc/azuredevops/client.go

```diff
@@ -107,9 +108,15 @@ func (c *client) do(ctx context.Context, req *http.Request, urlOverride string,
 	queryParams.Set("api-version", apiVersion)
 	req.URL.RawQuery = queryParams.Encode()
 	req.URL = u.ResolveReference(req.URL)
+	var reqBody []byte
 	if req.Body != nil {
 		req.Header.Set("Content-Type", "application/json")
+		reqBody, err = io.ReadAll(req.Body)
+		if err != nil {
+			return "", nil
```

Comment:
```suggestion
			return "", err
```

Don't we want to return the err if body cannot be read?

Reply:
Woops! Thanks